evaluators/text/EditDistanceEvaluator/tests/test_editdistanceevaluator.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from .. import EditDistanceEvaluator
+
+
+@pytest.mark.parametrize('actual, desired, distance',
```

Comment:
What happens when mixed strings of different lengths not empty and also capital letters?

Reply:
What if sentences with punctuation marks?